tests/Features/signup.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Signup
+
+    Scenario: Speaker successfully signs up
+        Given I am on the signup page
+        When I fill in the following:
+            | email | speaker@example.com |
```

Comment:
@mdwheele cool stuff, just one small note from the readability point of view.
You can add  more spaces in these "table" columns so it's easier to follow:
```
| email    | speaker@example.com |
| password | speaker             |

Reply:
Thanks @mheki! I ended up cooking this into the language itself rather than using the stock matchers from `MinkContext` (e.g. `When I fill in my account information`) captures the heart of behaviour without unnecessary detail